src/helpers/types.ts

```diff
@@ -186,7 +203,8 @@ export enum CardWebhookEvent {
   "CARD_AUTHORIZATION_DECLINE" = "CARD_AUTHORIZATION_DECLINE", // 	The transaction was declined.
   "CARD_AUTHORIZATION_RESOLUTION" = "CARD_AUTHORIZATION_RESOLUTION", // 	The reservation was cancelled, expired or booked. No push notification for the customer is required for this webhook.
   "CARD_LIFECYCLE_EVENT" = "CARD_LIFECYCLE_EVENT", // The status of the card is changed.
-  "CARD_TOKEN_LIFECYCLE" = "CARD_TOKEN_LIFECYCLE",
+  "CARD_TOKEN_LIFECYCLE" = "CARD_TOKEN_LIFECYCLE", // Push provisioning token is created or updated.
+  "CARD_FRAUD_CASE" = "CARD_FRAUD_CASE",
```

Comment:
I'm not sure about this one, we already have `pending` and `timeout` events

Reply:
🤦 it was a merging conflict that I solved incorrectly